### ISSUE TITLE
update gifsicle binary to 1.92 (lossy compression support)

### ISSUE THREAD

**Issue by mreinstein** [imagemin/gifsicle-bin#104]
Released yesterday, and adds lossy compression support to gifsicle. https://github.com/kohler/gifsicle/pull/72

**Comment by mreinstein** [comment]
@XhmikosR I don't mean to be a pest, but any chance we could get an updated gifsicle build? Right now I have to manually build gifsicle to get lossy compression support, and I'd much rather just pull in `gifsicle-bin` as an npm dependency.

**Comment by XhmikosR** [comment]
Sure, I just didn't notice sooner.

**Comment by XhmikosR** [comment]
Windows binaries added. The other platforms are pending which need someone else to provide the binaries.

**Comment by manojc** [comment]
Hello All,

Any updates on the binaries for other platforms?

@mreinstein Could you please let me know how can I do the manual build to import the latest binary?

**Comment by mreinstein** [comment]
Attached the instructions on how I did it. That said I don't understand why this isn't merged yet. This is an important feature. Most people that would want to run gifsicle for web image optimization want compression. To the powers that be, can we please merge this? Is there some holdup I'm not aware of on why we haven't?

## building gifsicle

spin up an amazon linux 64 bit x86 ec2 AMI  (x86 64 bit is super important, that ensures we're on the same platform that lambda runs on)

login to the ec2 instance and run this:

```bash
sudo yum update
sudo yum install autoconf intltool
sudo yum groupinstall "Development Tools"

wget https://github.com/kohler/gifsicle/archive/v1.92.tar.gz

tar -xzf v1.92.tar.gz
cd gifsicle-1.92

autoreconf -i
./configure --disable-gifview
make
```

then logout and run:

```bash
scp -i sp-tmp.pem ec2-user@18.233.161.219:/home/ec2-user/gifsicle-1.92/src/gifsicle /Users/mikereinstein/Sites/omni-asset-pipeline/gifsicle
```

**Comment by XhmikosR** [comment]
The problem is that I can't test this myself, otherwise I'd have merged it.

/CC @sindresorhus

**Comment by sindresorhus** [comment]
I've updated the macOS binary, but I don't currently have a Linux VM. If someone wants an updated Linux binary, submit a PR with a Docker script or something I can simply run to get the binary.

**Comment by mreinstein** [comment]
@sindresorhus I don't know much about docker, but if it would help I could create a .pem file, and give you access to an amazon ec2 server so that you could build and retrieve this binary. Would that suffice?

Just let me know where to send the .pem file (email perhaps?) And I'll happily send it along.

**Comment by 1000ch** [comment]
This is partially solved by 8635120.

**Comment by seahindeniz** [comment]
Is there any progress in adding new options such as `lossy` since #115 has merged?

> --lossy[=lossiness]
Alter image colors to shrink output file size at the cost of artifacts and noise. Lossiness determines how many artifacts are allowed; higher values can result in smaller file sizes, but cause more artifacts. The default lossiness is 20.
